src/components/page/FoodsPage.tsx

```diff
@@ -27,14 +27,14 @@ import React, { useEffect, useState } from 'react';
 export const FoodsPage = ({ containers }: { containers: IContainer[] }) => {
   const searchParams = useSearchParams();
   const sort = searchParams?.get('sort') || '';
-  const group = searchParams?.get('group') || '';
-  const container = searchParams?.get('container') || '';
+  const groupId = searchParams?.get('group') || '';
+  const containerId = searchParams?.get('container') || '';
```

Comment:
Clarified the variables by changing `group`/`container` to `groupId`/`containerId`